app/views/pages/index.html.erb

```diff
@@ -16,7 +16,6 @@
   </li>
 </ul>
 <hr/>
-<div id="content"></div>
-
-
-
+<div id="content">
+  <%= react_component('App') %>
```

Comment:
@justin808 I totally agree we will 2 separate tabs for client side rendering and server side rendering. The idea for this PR is for client side rendering.